automatic/jenkins/jenkins.nuspec

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+    <metadata>
+        <id>jenkins</id>
+        <version>2.222.4</version>
+        <packageSourceUrl>https://github.com/git@github.com:chocolatey-community/chocolatey-packages/tree/master/automatic/jenkins</packageSourceUrl>
+        <owners>pauby,johnypony3</owners>
+        <title>Jenkins CI</title>
+        <authors>Kohsuke Kawaguchi</authors>
+        <projectUrl>https://www.jenkins.io/</projectUrl>
+        <iconUrl>https://cdn.jsdelivr.net/gh/git@github.com:chocolatey-community/chocolatey-packages/icons/jenkins.png</iconUrl>
+        <copyright>2018 Kohsuke Kawaguchi</copyright>
```

Comment:
Is this still the current copyright?

Reply:
I'm unable to find a particularly good one for this.

On https://www.jenkins.io/license/, we see `Copyright (c) 2011, contributors of the Jenkins project`.
On a recent jenkins.exe we see some very different details (I suspect the EXE is just a service wrapper).
On https://github.com/jenkinsci/jenkins/blob/master/LICENSE.txt we see `Copyright (c) 2004-, Kohsuke Kawaguchi, Sun Microsystems, Inc., and a number of other of contributors`.
In Jenkins.xml, within a fresh install of Jenkins on a Windows box, we see `The MIT License Copyright (c) 2004-2017, Sun Microsystems, Inc., Kohsuke Kawaguchi, Oleg Nenashev, and other Jenkins contributors `. (again, very out of year)

I don't know where the 2018 Kohsuke Kawaguchi was sourced from originally. We could update it to say `2004-`?